google/cloud/forseti/services/cli.py

```diff
@@ -788,7 +788,7 @@ def do_create_model():
         """Create a model."""
         result = client.new_model('inventory',
                                   config.name,
-                                  config.inventory_index_id,
+                                  long(config.inventory_index_id),
```

Comment:
Python will auto-convert an int to a long if it's greater than maxint, yeah? So I don't think we need to specify long here, int is fine.

```
>>> sys.maxint
9223372036854775807
>>> x = sys.maxint
>>> x
9223372036854775807
>>> x = sys.maxint + 1
>>> x
9223372036854775808L
```

Reply:
Updated.